stable/memcached/templates/statefulset.yaml

```diff
@@ -17,6 +17,9 @@ spec:
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
+        {{- range $key, $val := .Values.customLabels }}
```

Comment:
Could you use `with` like podAnnotations below?

Reply:
@olemarkus I've added the requested changes, thanks for such a quick response!